src/emoji.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Text } from 'react-native';
+import nodeEmoji from 'node-emoji';
+
+PropTypes = require('prop-types');
```

Comment:
`react-native-emoji` does not support React 16.  Here we do support it.

Reply:
Awesome, you are right!